packages/language-server/src/completion/completions.json

```
@@ -442,6 +442,14 @@
     {
       "label": "sqlserver",
       "documentation": "Specifies a Microsoft SQL Server data source. Learn more about this connector [here](https://pris.ly/d/mssql-connector)"
+    },
+    {
+      "label": "mongodb",
+      "documentation": "Specifies a MongoDB data source. Learn more about this connector [here](https://pris.ly/d/mongodb-connector)"
+    },
+    {
+      "label": "cockroachdb",
+      "documentation": "Specifies a CockroachDB data source."
```

This will be the URL:
```suggestion
      "documentation": "Specifies a CockroachDB data source. Learn more about the CockroachDB data source connector [here](https://www.prisma.io/docs/reference/database-connectors/cockroachdb)"
```
Not sure we need to use pris.ly here. If we do, just update.

---

thanks!